tsconfig.json

```
@@ -1,16 +1,20 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
-  "extends": "@tsconfig/next/tsconfig.json",
   "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "isolatedModules": true,
+    "skipLibCheck": true,
     "noEmit": true,
     "strict": true,
     "esModuleInterop": true,
     "moduleResolution": "nodenext",
+    "module": "NodeNext",
     "allowJs": false,
     "target": "es2021",
     "declaration": true,
     "sourceMap": true,
     "newLine": "lf",
+    "jsx": "react",
```

Ideally we would use this, is it possible?
```suggestion
    "jsx": "react-jsx",
```

---

yep 👍 
33243f8